docs/wiki/Getting started - Setup environment.md

```diff
@@ -163,14 +163,14 @@ To use the environment's pipelines you should use the information you gathered d
 
 ### 3.2.2 Setup variables file
 
-The primary pipeline variable file `.github/variables/global.variables.json` hosts the fundamental pipeline configuration. In the file you will find and can configure settings such as:
+The primary pipeline variable file `global.variables.yml` hosts the fundamental pipeline configuration. In the file you will find and can configure settings such as:
```

Comment:
@eriqua, we should be aware this may require updates in the updated Wiki too

Reply:
Same for the other files like `PipelineDesign`

Reply:
Actually nevermind. This page here is already new. Then only in the `PipelineDesign`.

Reply:
@MrMCake Is there something need to action here?